src/viam/sdk/tests/test_camera.cpp

```diff
@@ -167,7 +167,7 @@ BOOST_AUTO_TEST_SUITE_END()
 
 BOOST_AUTO_TEST_SUITE(test_camera_client)
 
-MockCameraClient client = MockCameraClient("camera");
+MockCameraClient client("camera");
```

Comment:
I like this improvement, and I feel like there are several places in this review where the style is moving to be less prolix with types.